content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
-- In the **Tags** box, type "testing".
+In the next part of the tutorial, we will simulate some common use cases that someone would add annotations for.
 
-Adding annotations to your dashboards is a great way to communicate important events to the rest of your team.
+- To manually add an annotation, click anywhere in your graph, then click **Add annotation**.
+- In **Description**, enter **Migrated user database**.
+- In **Tags**, enter **migration**.
+- Click **Save**.
+  Grafana adds your annotation to the graph. Hover your mouse over the base of the annotation to read the text.
+
+((Marcus - Since we are having them tag the annotations, you might want to demonstrate how to use them, or at least explain what they do.))
+Grafana also lets you annotate a time interval, with _region annotations_.
+
+Add a region annotation.
 
-Manually annotating your dashboard is fine for those one-off events. For regularly occurring events, such as deploying a new release, Grafana supports querying annotations from one of your data sources:
+- Press Ctrl (or Cmd on macOS), then click and drag across the graph to select an area.
+- In **Description**, enter **Performed load tests**.
+- In **Tags**, enter **testing**.
+((In case you wondered, I'm changing "type" to "enter" because there are multiple ways to input the value. If we specify type, then they might think they cannot copy and paste. In reality, we don't care how they get the value there, so we say "enter."))
 
-- Select **Dashboard settings** from the top of the dashboard view.
-- Click **Annotations**, then **New Annotation Query**.
-- In the **Name** box, type "Errors".
-- Select "Loki" from the **Data source** drop down.
-- In the **Query** box, type a LogQL query:
+Manually annotating your dashboard is fine for those single events. For regularly occurring events, such as deploying a new release, Grafana supports querying annotations from one of your data sources. Let's create an annotation using the Loki data source we added earlier.
+((One hallmark of tutorials is friendly language like this. We hold their hand and walk them through what we are doing and why.))
+
+- At the top of the dashboard, click the **Dashboard settings** (gear) icon. ((It would be so much better if we could just embed small images of the icons in the text.))
+- In Annotations, click **Add Annotation Query**.
+- In **Name**, enter **Errors**.
+- In **Data source**, select **Loki**.
+- In **Query**, enter the following query:
 
 ```
 {filename="/var/log/tns-app.log"} |= "error"
 ```
 
-- Click **Add** and go back to your dashboard.
+- Click **Add**. Grafana displays the Annotations list, with your new annotation. 
+- Click the **Go back** arrow to return to your dashboard.
 
-The log lines returned by your query are now displayed as annotations in the graph.
+The log lines returned by your query are now displayed as annotations in the graph. ((Am I supposed to be seeing new annotations? I do not see any in my Traffic graph.))
```

Comment:
It seems like you didn't generate enough traffic on the sample application to produce enough errors. This is tricky, because the sample application produces errors randomly.